Makefile

```diff
@@ -334,7 +334,7 @@ check_perltidy:
 	test -z "${TO_TIDY_CHECK}" || ${DOCKER_COMPOSE} run --rm --no-deps backend perltidy --assert-tidy -opath=/tmp/ --standard-error-output ${TO_TIDY_CHECK}
 
 # same as check_perltidy, but this time applying changes
-lint_perltidy:
+lint:
```

Comment:
`lint` target already exists and depends `lint_perltidy` so you should not change this target name. The idea is that maybe at some point we want to add more lint commands.

Reply:
Ooh, I misunderstood Stephane 
`make lint` run directly all the "lint" commands